chat-plugins/nightclub.js

```diff
@@ -0,0 +1,154 @@
+//Nightclub by SilverTactic (Siiilver) and panpawn
+'use strict';
+
+if (!global.nightclubs) global.nightclubs = {};
```

Comment:
this isn't probably needed, and we can't just willy nilly make globals by just declaring them in the chat plugin